generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -228,13 +228,25 @@ def package_to_apkbuild(ros_distro, package_name,
                 if date is not None:
                     ver_suffix = '_git' + date
 
-    pkg.evaluate_conditions(os.environ)
-
     ros2_distros = [
         name for name, value in get_index(get_index_url()).distributions.items()
         if value.get("distribution_type") == "ros2"]
     is_ros2 = ros_distro in ros2_distros
 
+    # Make ROS_VERSION available in package.xml
+    orig_ros_version_env = os.environ.get("ROS_VERSION")
+    if is_ros2:
+        os.environ["ROS_VERSION"] = "2"
+    else:
+        os.environ["ROS_VERSION"] = "1"
+
+    pkg.evaluate_conditions(os.environ)
+
+    if orig_ros_version_env:
+        os.environ["ROS_VERSION"] = orig_ros_version_env
+    else:
+        del os.environ["ROS_VERSION"]
```

Comment:
Updated environment variable takes effect only in this process and child processes, so it's not needed to revert this.

It might be better to output error and exit if original ROS_VERSION is incorrect.

Reply:
>Updated environment variable takes effect only in this process and child processes, so it's not needed to revert this.

Done in https://github.com/alpine-ros/ros-abuild-docker/pull/165/commits/20d33e1f10468953c0305d9805ae2b785e210d87

>It might be better to output error and exit if original ROS_VERSION is incorrect.

With the change in https://github.com/alpine-ros/ros-abuild-docker/pull/165/commits/20d33e1f10468953c0305d9805ae2b785e210d87, original ROS_VERSION is no longer used. So, I don't think checking the original ROS_VERSION is needed.